gef.py

```diff
@@ -7016,16 +7023,20 @@ def context_regs(self):
             old_value = self.old_registers.get(reg, 0)
 
             padreg = reg.ljust(widest, " ")
-            line += "{}: ".format(Color.colorify(padreg, regname_color))
+            value = align_address(new_value)
+            old_value = align_address(old_value)
+            if value == old_value:
```

Comment:
On first run should these all be changed or the same? seems that they are all changes if you use `start` and some of them are changed if you do a normal break+run

Reply:
I don't care much about this. We default the old value to zero, so any register that isn't 0 will show as changed.

Reply:
Sure, just pointing it out :)